packages/@okta/vuepress-theme-prose/components/Quickstart.vue

```diff
@@ -0,0 +1,267 @@
+        activeClient: null,
+        activeClientComponent: null,
+        activeServer: null,
+        activeServerComponent: null,
+        activeFramework: null,
+        otpItems: [
+          {'title': 'Create an Account', 'path': '#create-an-okta-account'},
+          {'title': 'Client Setup', 'path': '#client-setup'},
+          {'title': 'Server Setup', 'path': '#server-setup'}
+        ]
+      }
+    },
+
+    beforeMount: function () {
+      if(window.location.hash == "") {
+        window.location.hash = this.defaultUriHash
+      }
+
+      this.currentHash = window.location.hash
+
+      const matches = window.location.hash.match('/([^/]*)?/?([^/]*)?/?([^/]*)');
+      this.activeClient = matches && matches[1];
+      this.activeServer = matches && matches[2];
+      this.activeFramework = matches && matches[3];
+
+
+    },
+
+    watch: {
+      currentHash: function () {
+        this.checkActiveClient()
+        this.checkActiveServer()
+
+        let clientComponent = this.$site.pages.find((page) => {
+          return page.regularPath.startsWith('/quickstart-fragments/'+this.activeClient+'/default-example')
+        })
+        if(clientComponent) {
+          this.activeClientComponent = clientComponent.key
+        }
+
+
+        let client = this.$themeConfig.quickstarts.clients.filter((client) => {
+          return client.name === this.activeClient
+        })[0];
+        let serverComponent = this.$site.pages.find((page) => {
+          return page.regularPath.startsWith('/quickstart-fragments/' + this.activeServer + '/' + this.activeFramework + '-' + client.serverExampleType + '/')
+        })
+        if(serverComponent) {
+          this.activeServerComponent = serverComponent.key
+        }
+
+
+      },
+
+      activeClient: function () {
+        window.location.hash = '/'+this.activeClient+'/'+this.activeServer+'/'+this.activeFramework
+        this.currentHash = window.location.hash
+      },
+
+      activeServer: function () {
+        let frameworkToSet = null
+        // is current Framework available for current Server
+        this.$themeConfig.quickstarts.servers.find((server) => {
+          if (server.name == this.activeServer) {
+            return server.frameworks.find((framework) => {
+              // console.log(framework.name, this.activeFramework)
+              if (framework.name == this.activeFramework) {
+                frameworkToSet = this.activeFramework
+                return
+              }
+            })
+          }
+        })
+        // get Default Framework for server if current one is not available
+        if (frameworkToSet === null) {
+          this.$themeConfig.quickstarts.servers.find((server) => {
+            if (server.name == this.activeServer) {
+              return server.frameworks.find((framework) => {
+                // console.log(framework.name, this.activeFramework)
+                if (framework.default && framework.default===true) {
```

Comment:
nit: one half of this is redundant - you only need either `if (framework.default) { ` or `if (framework.default === true) { `